docs/unified/msi/hcl.md

```diff
@@ -72,6 +67,7 @@ hardware.
         | 13th Gen Intel(R) Core(TM) i5-13600K | v0.9.0 | Dasharo HCL Report |
         | 13th Gen Intel(R) Core(TM) i5-13600K | v1.1.2 | Dasharo HCL Report |
         | 14th Gen Intel(R) Core(TM) i7-14700K | v0.9.0 | Dasharo HCL Report |
+        | Intel(R) Core(TM) i7-14700K | v0.9.0 | Dasharo HCL Report |
```

Comment:
There is something wrong here; there should be a 14th at the beginning for consistency.